content/backend/graphql-js/5-connecting-server-and-database.md

```diff
@@ -205,7 +204,6 @@ So, here's the trick. You're instantiating `Prisma` with the following pieces of
 
 - `typeDefs`: This points to the Prisma database schema which defines the full CRUD GraphQL API of Prisma. Note that you actually don't have this file yet - we'll tell you in a bit how to get it.
 - `endpoint`: This is the endpoint of your Prisma API. Don't forget to replace it with the endpoint of your own Prisma service here!
-- `secret`: Recall that all requests against the Prisma API need to be authenticated by including a JWT in the `Authorization` header of the HTTP request? This JWT needs to be signed with the `secret` defined in `prisma.yml`. As you're not making any _direct_ requests against the Prisma API, but these requests are being made for you by the `Prisma` binding instance, you need to tell it what that secret is so it can generate a JWT which it attaches to the requests.
```

Comment:
As I mentioned in the other review, I'd probably keep the `secret` in `prisma.yml` as it's very much best practice.